x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -18,6 +18,7 @@ export const ENDPOINT_ACTION_RESPONSES_INDEX = `${ENDPOINT_ACTION_RESPONSES_DS}-
 export const ENDPOINT_ACTION_RESPONSES_INDEX_PATTERN = `${ENDPOINT_ACTION_RESPONSES_DS}-*`;
 
 export const eventsIndexPattern = 'logs-endpoint.events.*';
+export const fileEventsIndexPattern = 'logs-endpoint.events.file-*';
```

Comment:
Suggestion: this is a constant - can you use all uppercase-snake casing naming for the variable?